doc/releases/changelog-dev.md

```diff
@@ -12,11 +12,32 @@
   gate-set that can be translated to the MBQC formalism.
   [(7271)](https://github.com/PennyLaneAI/pennylane/pull/7271)
 
+<h4>Resource-efficient Decompositions 🔎</h4>
+
+* The `decompose` transform now supports symbolic operators specified as strings in the gate set
+  when the new graph-based decomposition system is enabled.
+  [(#7331)](https://github.com/PennyLaneAI/pennylane/pull/7331)
+
+  ```python
+  from functools import partial
+  import pennylane as qml
+  
+  qml.decomposition.enable_graph()
+   
+  @partial(qml.transforms.decompose, gate_set={"T", "Adjoint(T)", "H", "CNOT"})
+  def circuit():
+      ...
+  ```
+
 <h3>Improvements 🛠</h3>
 
 * Alias for Identity (`I`) is now accessible from `qml.ops`.
   [(#7200)](https://github.com/PennyLaneAI/pennylane/pull/7200)
 
+* The `gate_set` argument of `qml.transforms.decompose` accepts `"X"`, `"Y"`, `"Z"`, `"H"`, `"I"` as
```

Comment:
@isaacdevlugt Any concerns here?

Reply:
Is it _just_ those operators? 🤔 

```suggestion
* The `gate_set` argument of :func:`~.transforms.decompose` now accepts `"X"`, `"Y"`, `"Z"`, `"H"`, `"I"` as
```

Reply:
Yes, these are the only operators we have aliases for, unless I'm missing anything. The nice thing is that we also support things like `"Adjoint(H)"` which gets translated to `"Adjoint(Hadamard)"`

Reply:
Gotcha! Only suggestion is to add an RTD link :)